app/ForkedStar.tsx

```diff
@@ -0,0 +1,73 @@
+// const { Octokit } = require("@octokit/action");
+const { Octokit } = require("octokit"); // Use this for local development
+import { env } from "@/env.mjs";
+import { GoRepoForked } from "react-icons/go";
+import { GoStar } from "react-icons/go";
+
+const GITHUB_TOKEN = env.GITHUB_TOKEN;
+const GITHUB_ORG = env.GITHUB_ORG;
+
+const octokit = new Octokit({
+  auth: GITHUB_TOKEN,
+});
```

Comment:
Not required when using the pre-authenticated octokit instance. See how it's done in other places. Eg https://github.com/coronasafe/leaderboard/blob/main/app/feed/page.tsx#L18

```suggestion
const GITHUB_ORG = env.GITHUB_ORG;
```